models/upload_sessions.go

```diff
@@ -187,7 +190,13 @@ func (u *UploadSession) StartUploadSession() (vErr *validate.Errors, err error)
 		DB.ValidateAndUpdate(u)
 	}
 
-	vErr, err = BuildDataMaps(u.GenesisHash, u.NumChunks)
+	go func() {
+		vErr, err = BuildDataMaps(u.GenesisHash, u.NumChunks)
+	}()
+	// TODO:  Remove this.  This is just to help make these changes in
+	// fewer PRs.  We may need to use WaitForAllChunks in some of our tests.
+	_, err = u.WaitForAllChunks(MaxTimesToCheckForAllChunks)
```

Comment:
This is still blocking. Shouldn't this also go in the goroutine?

Reply:
It is intentionally blocking, for now.  This is to be able to handle the various async changes in smaller PRs.

Reply:
This will be pulled out in another PR later today or tomorrow.